dts/bindings/sensor/nxp,fxas21002-common.yaml

```diff
@@ -3,11 +3,18 @@
 
 description: FXAS21002 3-axis gyroscope sensor
 
-compatible: "nxp,fxas21002"
-
-include: [sensor-device.yaml, i2c-device.yaml]
+include: sensor-device.yaml
 
 properties:
+    reset-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        RST pin
+        This pin defaults to active high when consumed by the sensor.
```

Comment:
"active high", but you set active low just above, is this correct?

Reply:
Changed it to  - "This pin defaults to active low when consumed by the sensor"